api/src/main/resources/workbench-api.yaml

```diff
@@ -2191,6 +2191,50 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/FileDetail'
+  /v1/admin/workspaces/{workspaceNamespace}/runtimes:
+    get:
+      tags:
+        - workspaceAdmin
+      description: Get a list of runtimes in this Workspace. RESEARCHER_DATA_VIEW authority required.
+      operationId: listRuntimes
```

Comment:
nit: not sure the naming conversion here, but we might want to consider use a different operation Id like `adminListRuntimesInWorkspace` so that we can easily tell the difference among "user listRuntimes in all workspaces", "user listRuntimes in the workspace" "admin list in all workspaces", "admin list in the workspace"

Reply:
same comment for userApp below

Reply:
good call.  changed.